src/app/api-doc/ReactSwagger.tsx

```diff
@@ -0,0 +1,14 @@
+'use client';
```

Comment:
could you rename `api-doc` to `api_doc`?

Reply:
Why do we need to split this component?

Reply:
> could you rename `api-doc` to `api_doc`?

ok I will


> Why do we need to split this component?

In order to use  this component as client component or they won't work within this Server Components 
read more: https://beta.nextjs.org/docs/rendering/server-and-client-components#moving-client-components-to-the-leaves

Reply:
How's about use client in page.tsx